src/mpid/ch4/src/ch4_coll_globals_default.c.in

```diff
@@ -24,6 +24,12 @@ const MPIDI_coll_algo_container_t CH4_Barrier_intra_composition_beta_cnt = {
                                                    .barrier = -1}
 };
 
+const MPIDI_coll_algo_container_t CH4_Barrier_intra_composition_gamma_cnt = {
+    .id = MPIDI_CH4_Barrier_intra_composition_gamma_id,
+    .params.ch4_barrier_params.ch4_barrier_gamma = {
+                                                    .barrier = -1}
```

Comment:
Would be great if we could have some special define instead of magic number. MPIDI_COLL_SELECT_AUTO or something like that.